client/packages/system/src/Name/ListView/Stores/StoreProperties.tsx

```diff
@@ -125,3 +129,17 @@ const Row = ({
     </Box>
   );
 };
+
+const translatePropertyLabels = (
+  t: TypedTFunction<LocaleKey>,
+  label: string
+) => {
+  // For now, the rest of the properties have to be initialised, and are saved
+  // in french so have not included translations yet
+  switch (label) {
+    case 'Packaging Level':
+      return t('label.packaging-level');
```

Comment:
Yo just popping in, was a bit confused what you were meaning in stand up about only translating some of the properites, but I think this makes sense in that it should always ship with OMS, rather than only if GAPS is initialised...

What if we just store `label.packaging-level` in the DB, and call `t()` on it here? `t` would return the base label if no translation exists (for the existing property keys) but that way we can see from data in the DB, its a translation string, rather than looking like english but showing in french?

Reply:
That would work too... but the other GAPS properties will also need to be changed..